kin-sdk-core/src/androidTest/java/kin/sdk/core/EncryptorImplV23Test.java

```diff
@@ -0,0 +1,50 @@
+package kin.sdk.core;
+
+import static junit.framework.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assume.assumeTrue;
+
+import android.os.Build;
+import android.os.Build.VERSION_CODES;
+import org.junit.Before;
+import org.junit.Test;
+
+public class EncryptorImplV23Test {
```

Comment:
check if you can change emulator android version that runs tests

Reply:
I have additional problem with android tests, will investigate it in another task